File: AEDS II/codigos/0-apresentacao-linguagens/ParImpar.java
import java.util.Scanner;

public class ParImpar{
  public static void imprimirParImpar(int valor){
    if(valor % 2 == 0){ System.out.println("1");}
    else{ System.out.println("2");}
  }

  public static void main(String[] args){
    int valor, valorDigitado;
    
    Scanner teclado = new Scanner(System.in);
    
    valor = teclado.nextInt();

    for(int i = 0; i < valor; i++){
      valorDigitado = teclado.nextInt();
      imprimirParImpar(valorDigitado);
    }
  }
}

File: AEDS II/codigos/1 - parImpar/parImpar.c
#include <stdio.h>

// Método com Linker
// void parImpar(int valor);

void parImpar(int valor){
  if(valor % 2 == 0){ printf("1 \n"); }
  else{  printf("2 \n"); }
}

int main(){
  int valor, valorDigitado;
  scanf("%d", &valor);

  for(int i = 0; i < valor; i++){
    scanf("%d", &valorDigitado);
    parImpar(valorDigitado);
  }

  return 0;
}

// Método do Linker
/*
void parImpar(int valor){
  if(valor % 2 == 0){printf("1 \n");}
  else{printf("2 \n");}
}
*/
